sites/public/pages/index.tsx

```diff
@@ -82,7 +82,7 @@ export async function getStaticProps() {
   let listings = []
   try {
     const response = await axios.get(process.env.listingServiceUrl)
-    listings = response.data
+    listings = response.data.items ? response.data.items : null
```

Comment:
You have `data.items ?` here and `data ? data.items : []` above. Could there exist a case where data is valid but data.items is not?

Reply:
I don't think so, it looks like `items` is a required property of the `Pagination` object. Will update to check `response.data` here.